lib/widgets/toolbar.dart

```diff
@@ -353,12 +353,12 @@ Widget defaultToggleStyleButtonBuilder(
   final theme = Theme.of(context);
   final isEnabled = onPressed != null;
   final iconColor = isEnabled
-      ? isToggled != null
+      ? isToggled == true
```

Comment:
Nit: You could omit `true`.
```suggestion
      ? isToggled
```

Reply:
We can only do this with this nullable variable
```dart
void main() {
  bool? isToggled = null;
  print(isToggled!); //Script error.
}
```

Reply:
void main() {
  bool? isToggled = null;
  print(isToggled); //null
}

Reply:
Thanks I understand, I haven't had a chance to look at non-nullable yet.